src/app/auth/AuthRequestHandler.ts

```diff
@@ -192,7 +192,7 @@ class Organisation implements User {
   constructor(kvk: string, userName: string, config: UserConfig) {
     this.kvk = kvk;
     this.identifier = kvk;
-    this.userName = userName;
+    this.userName = userName ?? kvk;
```

Comment:
Username mag niet helemaal niet undefined zijn van de typechecker?

Reply:
Nee van de typechecker niet, maar zie hier hoe typescript een leaky abstraction is ;).